golem_messages/constants.py

```diff
@@ -44,3 +45,8 @@
         message.concents.ForceReportComputedTask: (2 * MMTT + MAT),
     },
 )
+
+
+class MarketType(str, Enum):
+    BRASS_MARKET = "brass"
```

Comment:
Is the '_MARKET' suffix required? Please consider using `datastructures.StringEnum` and style similar to
https://github.com/golemfactory/golem-messages/blob/051492623d99135ef6928abe9791062d8105406f/golem_messages/message/tasks.py#L503-L508